data/phi0474/phi031/phi0474.phi031.perseus-lat1.xml

```diff
@@ -50,6 +50,12 @@
 		</fileDesc>
 
 		<encodingDesc>
+			<refsDecl n="CTS">
+				<cRefPattern matchPattern="(\w+)"
+					replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])">
+					<p>This pointer pattern extracts section</p>
+				</cRefPattern>
+			</refsDecl>
 			<refsDecl>
```

Comment:
All of the old refState stuff seems unnecessary. We either strike or comment out.